slides/linux_commands.qmd

```diff
@@ -0,0 +1,81 @@
+# Basic Linux Commands {.smaller}
+
+## `cd`: Navigate Directories {.smaller}
+- `cd [directory]` -- Changes the current working directory to the specified directory.
+- `cd ..` -- Move up one directory level.
+- `cd /path/to/directory` -- Go to a specific path.
+- `cd ~` or `cd` -- Go to the home directory.
+- `cd /` -- Go to the root directory.
+
+## `ls`: List Directory Contents {.smaller}
+- `ls` -- Lists files and directories in the current working directory.
+- `ls -l` -- Displays detailed information about each file (permissions, owner, size, etc.).
+- `ls -a` -- Shows all files, including hidden files (files starting with a dot).
+- `ls -h` -- Displays sizes in human-readable format (e.g., KB, MB).
+- `ls -lah` -- Combines the above options.
+
+
+## `mkdir`: Make Directory {.smaller}
+- `mkdir [directory_name]` -- Creates a new directory with the specified name.
+- `mkdir -p /path/to/dir` to create nested directories.
+
+
+## `rmdir` and `rm`: Remove Directories and Files {.smaller}
+- `rm [file] `-- Removes a file.
+- `rmdir [directory]` -- Removes an empty directory.
+- `rm -r [directory]` - Removes a directory and its contents recursively.
+- `rm -f [file]` - Removes to force remove a file (no undo be careful!).
+
+
+## `mv` and `cp`: Move, Rename and Copy {.smaller}
+- `cp [source] [destination]` -- Copies a file or directory to the destination.
+- `mv [source] [destination]` -- Moves a file or directory to the destination.
+- `mv` can also be used to rename a file or directory if the source and destination directories match.
+
+```{bash}
```

Comment:
The way these code cells are specified, I think they'll be executed at build time.
You can use `.bash` if you wish them to render with the right syntax highlighting but not execute.